src/views/Vacancies/components/FilterSidebar.tsx

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import { TextField, Typography, InputAdornment } from "@material-ui/core";
+import SearchIcon from "@material-ui/icons/Search";
+import CloseIcon from "@material-ui/icons/Close";
+
+import FacetFilter from "./FacetFilter";
+
+const useStyles = makeStyles(theme => ({
+  root: {
```

Comment:
I think sidebar components will be used in several places. How can we make sure we normalize styles for all?

Reply:
I think we can figure that out once we get final designs ready based on material ui.